functions.ipynb

```diff
@@ -402,6 +412,104 @@
     "print(greet(\"John\"))"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "28ee880e-0ecd-44b1-b9e2-07865a21a7da",
+   "metadata": {},
+   "source": [
+    "---"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "336848ef-90e1-4149-9640-b22eea872f09",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%reload_ext tutorial.tests.testsuite"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "468fad87-0005-4278-ba12-4caf0f43af86",
+   "metadata": {},
+   "source": [
+    "## Exercise 1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2779a92e-6aa2-49a4-b950-164fba120edd",
+   "metadata": {},
+   "source": [
+    "Write a Python function called `greet` that takes two parameters: `name` (a string) and `age` (an integer).\n",
+    "The function should return a greeting message in the following format: `\"Hello, <name>! You are <age> years old.\"`\n",
+    "\n",
+    "**Do not** forget to write a proper docstring and add the correct type hints to the parameters and the return value."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5212084f-9742-4f29-b3be-132e361eaee6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%ipytest\n",
+    "\n",
+    "def solution_greet():\n",
+    "    return"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7f19f0ba-08de-4717-8f08-7255bed0bcfd",
+   "metadata": {},
+   "source": [
+    "## Exercise 2\n",
+    "\n",
+    "Write a Python function called `calculate_area` that takes three parameters: `length` (a float), `width` (a float), and `unit` (a string with a **default** value of `\"cm\"`).\n",
+    "The function should calculate the area of a rectangle based on the given length and width, and return the result **as a tuple** with the correct, default unit (i.e., `cm^2`).\n",
+    "If the unit parameter is \"m\", the function should convert the length and width from meters to centimeters before calculating the area.\n",
+    "\n",
+    "Your solution function **must** handle the following input units (the output unit is **always** `cm^2`):\n",
+    "\n",
+    "- centimeters (`cm`)\n",
+    "- meters (`m`)\n",
+    "- millimeters (`mm`)\n",
+    "- yards (`yd`)\n",
```

Comment:
Should we provide the values for yards and feet to do the conversion?

Reply:
Mmm, I would say "no". It's fairly easy to look it up online. If you feel it's too much (😄), then feel free to add the conversion factors to the problem's text.

Reply:
Please see my requested changes below - I added a hint